sdk/monitor/azure-monitor-opentelemetry-exporter/src/test/java/com/azure/monitor/opentelemetry/exporter/AzureMonitorTraceExporterTest.java

```diff
@@ -38,10 +40,14 @@
 
     @Test
     public void testExportRequestData() {
+        String connectionStringTemplate = "InstrumentationKey=ikey;IngestionEndpoint=https://testendpoint.com";
+        String connectionString = Configuration.getGlobalConfiguration()
+            .get("APPLICATIONINSIGHTS_CONNECTION_STRING", connectionStringTemplate);
         AzureMonitorTraceExporter azureMonitorTraceExporter = getClientBuilder()
-            .connectionString("InstrumentationKey=ikey;IngestionEndpoint=https://testendpoint.com")
+            .connectionString(connectionString)
             .buildTraceExporter();
         CompletableResultCode export = azureMonitorTraceExporter.export(Collections.singleton(new RequestSpanData()));
+        export.join(30, TimeUnit.SECONDS);
```

Comment:
Is this added during "RECORD". can we remove this?

Reply:
This is a timeout value. So, this is the max it waits but in PLAYBACK mode, it doesn't have to wait for the response and completes immediately.